sources/models/src/aws-dev/mod.rs

```diff
@@ -5,7 +5,7 @@ use std::collections::HashMap;
 use crate::modeled_types::Identifier;
 use crate::{
     AwsSettings, BootstrapContainer, HostContainer, KernelSettings, MetricsSettings,
-    NetworkSettings, NtpSettings, UpdatesSettings,
+    NetworkSettings, NtpSettings, PEMCertificate, UpdatesSettings,
```

Comment:
If NTP is "Ntp" and AWS is "Aws" then PEM should probably be "Pem":
```suggestion
    NetworkSettings, NtpSettings, PemCertificate, UpdatesSettings,
```